### PR TITLE
Extend and fix tests.

### PR DESCRIPTION
**Pull request by mbarnach**
Test separately the emphasis and line break.
Add a test for the custom attributes.
Fix a typo related to custom attributes in the README.

### CODE REVIEW

File: Tests/VauxTests/VauxTests.swift [hunk 1]
```diff
@@ -80,6 +79,72 @@ final class VauxTests: XCTestCase {
       XCTFail(error.localizedDescription)
     }
   }
+    
+  func testLinebreak() {
+    var url = "https://google.com"
```
Comment (dokun1) [inline Tests/VauxTests/VauxTests.swift]: Can we remove this? If we are not using it in the test, we don't need it.

File: Tests/VauxTests/VauxTests.swift [hunk 2]
```diff
@@ -80,6 +79,72 @@ final class VauxTests: XCTestCase {
...
+        let rendered = try renderForTesting(with: vaux, html: pageWithLink())
+        // TODO: Make this pass with better string comparisons
+        XCTAssertEqual(rendered, correctHTML)
+    } catch let error {
+        XCTFail(error.localizedDescription)
+    }
+  }
+    
+  func testEmphasis() {
+    var url = "https://google.com"
```
Comment (dokun1) [inline Tests/VauxTests/VauxTests.swift]: Same thing here - the `var` only exists if we integrate it into a test.

File: Tests/VauxTests/VauxTests.swift [hunk 3]
```diff
@@ -292,6 +355,32 @@ final class VauxTests: XCTestCase {
       XCTFail(error.localizedDescription)
     }
   }
+    
+  func testAttributes() {
+    func buildPage() -> HTML {
+        html {
+            div {
+                "Custom tag text goes here"
+                }.attr("key", "value")
```
Comment (dokun1) [inline Tests/VauxTests/VauxTests.swift]: Can you please fix the indentation of the closing bracket to be aligned with the `div` here? Also, this project uses 2 spaces for tabs, so please adjust all of your code accordingly.

Comment (dokun1) [reply Tests/VauxTests/VauxTests.swift]: Additionally, feel free to test out some of the other strongly-typed attributes here, such as `class` or `id`.

**Review by dokun1 — CHANGES_REQUESTED** [review]

**Comment by mbarnach** [comment]
I've fix the indentation (I think all over, by changing my settings).
I've extended the test for the custom tag with the id and class attribute.

**Review by dokun1 — APPROVED** [review]

**Comment by dokun1** [comment]
🎉 thank you!

**Pull request merged**